scripts/upgrade

```diff
@@ -137,7 +137,9 @@ ynh_script_progression --message="Building app..."
 
 pushd "$final_path"
     ynh_use_nodejs
-    ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH yarn cleanall
+    	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH yarn run clean-all
+	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH sudo corepack enable
+	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH yarn set version berry
```

Comment:
```suggestion
        ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH corepack prepare yarn@stable --activate
	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH yarn set version berry	
```
Per https://yarnpkg.com/getting-started/install, section Updating the global Yarn version, I am wondering if we should do this command too.